hc/settings.py

```diff
@@ -135,47 +139,48 @@
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
-# Slack integration -- override these in local_settings
+# Slack integration -- override these in dev_settings
 SLACK_CLIENT_ID = None
 SLACK_CLIENT_SECRET = None
 
-# Pushover integration -- override these in local_settings
+# Pushover integration -- override these in dev_settings
 PUSHOVER_API_TOKEN = None
 PUSHOVER_SUBSCRIPTION_URL = None
 PUSHOVER_EMERGENCY_RETRY_DELAY = 300
 PUSHOVER_EMERGENCY_EXPIRATION = 86400
 
-# Pushbullet integration -- override these in local_settings
+# Pushbullet integration -- override these in dev_settings
 PUSHBULLET_CLIENT_ID = None
 PUSHBULLET_CLIENT_SECRET = None
 
-if os.path.exists(os.path.join(BASE_DIR, "hc/local_settings.py")):
-    from .local_settings import *
+# Parse database configuration from $DATABASE_URL
+import dj_database_url
+
+DATABASES['default'] = dj_database_url.config()
+
+
+if os.path.exists(os.path.join(BASE_DIR, "hc/dev_settings.py")):
```

Comment:
Dev settings is an optional file to change the default settings. It varies across developers. That is why I gitignored it.